File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc5-aspnet.identity-spa/csharp/App_Code/CaptchaHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using BotDetect.Web.Mvc;
namespace AspNetMvc523SinglePageApplicationExampleCSharp.App_Code
{
    public static class CaptchaHelper
    {
        public static MvcCaptcha GetExampleCaptcha(string captchaId)
        {
            // create the control instance
            MvcCaptcha exampleCaptcha = new MvcCaptcha(captchaId);

            // set up client-side processing of the Captcha code input textbox
            exampleCaptcha.UserInputID = "CaptchaCode";

            // Captcha settings
            exampleCaptcha.ImageSize = new System.Drawing.Size(255, 50);

            return exampleCaptcha;
        }
    }
}

File: bin/Debug/net5.0/v4.4.2/lgcy-on-lgcy/examples/t_api-captcha-mvc5-aspnet.identity-spa/vbnet/App_Code/CaptchaHelper.vb
Imports BotDetect.Web.Mvc
Public Class CaptchaHelper

    Public Shared Function GetExampleCaptcha(captchaId As String) As MvcCaptcha

        ' create the control instance
        Dim exampleCaptcha As MvcCaptcha = New MvcCaptcha(captchaId)

        ' set up client-side processing of the Captcha code input textbox
        exampleCaptcha.UserInputID = "CaptchaCode"

        ' Captcha settings
        exampleCaptcha.ImageSize = New System.Drawing.Size(255, 50)

        Return exampleCaptcha

    End Function
End Class
